qa/L0_batch_input/batch_input_test.py

```diff
@@ -58,17 +50,26 @@ def callback(user_data, result, error):
 
         self.client_callback = callback
 
+    def set_inputs(self, shapes, input_name):
+        self.dtype_ = np.float32
+        self.inputs = []
+        if len(shapes[0]) == 1:
+            for shape in shapes:
+                self.inputs.append(
+                    [grpcclient.InferInput(input_name, [1, shape[0]], "FP32")])
+                self.inputs[-1][0].set_data_from_numpy(
+                    np.full([1, shape[0]], shape[0], np.float32))
+        else:
```

Comment:
For future-proof, adding new test using this helper may result in unexpected input shape if not looking into the function definition.

Maybe better if breaking it into two set_input functions, i.e. `set_input` and `set_input_for_batch_item`

Reply:
Good idea. Done!